tests/ensemble_evaluator/test_ensemble_client.py

```diff
@@ -0,0 +1,83 @@
+import threading
+import pytest
+import asyncio
+import websockets
+from functools import partial
+from ert_shared.ensemble_evaluator.prefect_ensemble.client import Client
+
+
+def _mock_ws(host, port, messages, delay_startup=0):
```

Comment:
Would it be possible to merge this function with the one from `test_prefect_ensemble` and move it to `tests/ensemble/evaluator/conftest.py`?

Reply:
It would. Will it be done? That is another question, to which I can only reply, yes.